snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -105,7 +105,7 @@ def mount_project(self) -> None:
         # multipass keeps the mount active, so check if it is there first.
         if not self._instance_info.is_mounted(project_mountpoint):
             self._mount(
-                mountpoint=project_mountpoint, dev_or_path=self.project.project_dir
+                mountpoint=project_mountpoint, dev_or_path=self.project._project_dir
```

Comment:
Maybe I'm being silly, but... is this actually set somewhere? I don't see it being saved as an attribute in `Project`, nor do I see a property.

Reply:
a git add was missed!